### ISSUE TITLE
Semantically Markup Questions as <dl>?

### ISSUE THREAD

**Issue by jpdevries** [Thinkful-Ed/quizful#34]
FAQs, Question and Answer Lists, and of course definition lists are appropriate uses of the `<dl>` tag.

I think it is worth considering changing

``` jsx
<div className="question">
  <p>Question {this.state.currentQ + 1} of {this.state.questions.length}</p>
  <h1>{this.state.questions[this.state.currentQ].question}</h1>
  <ul className="answers">
    {allAnswers}
  </ul>
  <button className="prev-button" type="button" onClick={this.lastQuestion}>Previous Question</button>
  <button className="next-button" type="button" onClick={this.nextQuestion}>Next Question</button>
</div>
</section>
```

to

``` jsx
<div className="question">
  <p>Question {this.state.currentQ + 1} of {this.state.questions.length}</p>
  <dl>
    <dt><h1>{this.state.questions[this.state.currentQ].question}</h1></dt>
    <dd>
      <ul className="answers">
        {allAnswers}
      </ul>
    </dd>
  </dl>
  <button className="prev-button" type="button" onClick={this.lastQuestion}>Previous Question</button>
  <button className="next-button" type="button" onClick={this.nextQuestion}>Next Question</button>
</div>
</section>
```

Where as the question is the term and the answers are the potential definitions.

http://www.html-5-tutorial.com/dl-tag.htm
http://www.w3schools.com/TAGS/tag_dl.asp
https://www.marcozehe.de/2015/12/14/the-web-accessibility-basics/#lists

**Comment by aric87** [comment]
Definitely a interesting argument here. Not sure I agree/disagree really. I wonder if there is a precedent set somewhere for how a quiz should be setup? There has to be some sort of thing like this elsewhere, and I'd love to see how they do it.

**Comment by jpdevries** [comment]
The first link mentions that it `<dl>` is great for FAQs, which I think is similar to a quiz.
If you don't use use `<dl>` you'd have to use ARIA to simulate semantics.
